circle.yml

```diff
@@ -1,30 +0,0 @@
-version: 2
-executorType: machine
-stages:
-  build:
-    workDir: ~/gradle-docker
-    environment:
-      TERM: dumb
-    steps:
-      - type: checkout
-      - type: shell
-        shell: /bin/bash
-        command: ./gradlew --parallel -i build
-      - type: test-results-store
-        path: build/test-results
-      - type: deploy
-        shell: /bin/bash
-        command: ./gradlew -i bintrayUpload
-      - type: deploy
-        shell: /bin/bash
-        command: ./gradlew -i -Dgradle.publish.key=$GRADLE_KEY -Dgradle.publish.secret=$GRADLE_SECRET publishPlugins
```

Comment:
we need to add some code in build.gradle to set these:

https://github.com/palantir/sls-packaging/blob/2c285e6525f62632525c8f395e7240ea6cb0363c/gradle-sls-packaging/build.gradle#L83-L86